opencti-platform/opencti-front/src/private/components/analyses/reports/ReportEditionOverview.jsx

```diff
@@ -199,6 +204,8 @@ const ReportEditionOverviewComponent = (props) => {
       enableReinitialize={true}
       initialValues={initialValues}
       validationSchema={reportValidator}
+      validateOnChange={false} // Validation will occur on submission, required fields all have *'s
+      validateOnBlur={false} // Validation will occur on submission, required fields all have *'s
```

Comment:
On Edition form, we need to set validateOnChange and validateOnBlur to `true` because we don't have  submit button, submit is onBlur.